src/garage/envs/multi_env_wrapper.py

```diff
@@ -194,52 +265,42 @@ def _active_task_one_hot(self):
         one_hot[index] = self.task_space.high[index]
         return one_hot
 
-    def reset(self, **kwargs):
-        """Sample new task and call reset on new task env.
+    def __getattr__(self, attr):
```

Comment:
does this preserve the documentation for the methods/attributes it passes through?